### PR TITLE
[Improve] [bin] Remove redundant shell for install-plugin.sh

### PR DESCRIPTION
**Pull request by TaoZex**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by hailin0** [comment]
@CalvinKirs

**Review by TyrantLucifer — DISMISSED** [review]

File: bin/install-plugin.sh [hunk 1]
```diff
@@ -43,57 +43,16 @@ if [ ! -d ${SEATUNNEL_HOME}/connectors ];
...
-
 # create the seatunnel connectors directory (for v2)
 if [ ! -d ${SEATUNNEL_HOME}/connectors/seatunnel ];
   then
       mkdir ${SEATUNNEL_HOME}/connectors/seatunnel
       echo "create seatunnel connectors directory"
 fi  
 
-path=flink-sql
+path=seatunnel
```
Comment (liugddx) [inline bin/install-plugin.sh]: This variable can be removed

Comment (TaoZex) [reply bin/install-plugin.sh]: Done. Thanks for your advice.

File: bin/install-plugin.sh [hunk 2]
```diff
@@ -43,61 +43,18 @@ if [ ! -d ${SEATUNNEL_HOME}/connectors ];
...
-	  # v2 connectors
-	  if [ "$line" = "--connectors-v2--" ]
-	    then
-	  	 path=seatunnel
-	  fi
     if  [ ${line:0:1} != "-" ] && [ ${line:0:1} != "#" ]
       	then
       		echo "install connector : " $line
-      		${SEATUNNEL_HOME}/mvnw dependency:get -DgroupId=org.apache.seatunnel -DartifactId=${line} -Dversion=${version} -Ddest=${SEATUNNEL_HOME}/connectors/${path}
+      		${SEATUNNEL_HOME}/mvnw dependency:get -DgroupId=org.apache.seatunnel -DartifactId=${line} -Dversion=${version} -Ddest=${SEATUNNEL_HOME}/connectors
```
Comment (liugddx) [inline bin/install-plugin.sh]:
```suggestion
      		${SEATUNNEL_HOME}/mvnw dependency:get -DgroupId=org.apache.seatunnel -DartifactId=${line} -Dversion=${version} -Ddest=${SEATUNNEL_HOME}/connectors/seatunnel
```

**Review by liugddx — COMMENTED** [review]
Connector Jar Dir need engine name

File: bin/install-plugin.sh (hunk 2 above)
Comment (TyrantLucifer) [reply bin/install-plugin.sh]: Aggre with @liugddx

File: seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples/spark.batch.conf [hunk 3]
```diff
@@ -1,98 +1,35 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
```
Comment (TyrantLucifer) [inline seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples/spark.batch.conf]: revert

**Review by TyrantLucifer — CHANGES_REQUESTED** [review]

File: seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples/spark.batch.conf [hunk 4]
```diff
@@ -1,98 +1,35 @@
-#
```
Comment (liugddx) [inline seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples/spark.batch.conf]: Why change this?

**Review by liugddx — APPROVED** [review]
LGTM

File: seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples/spark.batch.conf (hunk 4 above)
Comment (TaoZex) [reply seatunnel-examples/seatunnel-spark-connector-v2-example/src/main/resources/examples/spark.batch.conf]: Sorry, I submitted the wrong document.

**Review by TyrantLucifer — APPROVED** [review]

**Pull request merged**